terminfinder-chart/charts/terminfinder-frontend/templates/configmap.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: terminfinder-frontend
+  annotations:
+    use-subpath: "true"
+data:
+  customer-logo.svg: {{ .Values.customerConfig.logo | toYaml | nindent 4 }}
```

Mir wird vorgeschlagen den Wert zu quoten

```suggestion
  customer-logo.svg: {{ .Values.customerConfig.logo | toYaml | nindent 4 | quote }}
```

---

Gibt deine IDE eine Begründung aus? Also meiner Meinung nach ist das unnötig in Kombination mit `toYaml`.

Ich hab das auch mal ausprobiert und das Ergebnis ist Dasselbe...

---

Beim genauerer Betrachtung, werde ich es doch lieber quoten.

---

Hab es wieder rückgängig gemacht.